src/main/flight/gps_rescue.c

```diff
@@ -640,28 +640,28 @@ void disarmOnImpact(void)
 void descend(void)
 {
     if (newGPSData) {
-        const float distanceToLandingAreaM = rescueState.sensor.distanceToHomeM - (rescueState.intent.targetLandingAltitudeCm / 200.0f);
+        const float distanceToLandingAreaM = rescueState.sensor.distanceToHomeM - (rescueState.intent.targetLandingAltitudeCm * 0.005f);
         // considers home to be a circle half landing height around home to avoid overshooting home point
         rescueState.intent.proximityToLandingArea = constrainf(distanceToLandingAreaM / rescueState.intent.descentDistanceM, 0.0f, 1.0f);
         rescueState.intent.velocityPidCutoffModifier = 2.5f - rescueState.intent.proximityToLandingArea;
          // 1.5 when starting descent, 2.5 when almost landed; multiplier for velocity step cutoff filter
         rescueState.intent.targetVelocityCmS = gpsRescueConfig()->rescueGroundspeed * rescueState.intent.proximityToLandingArea;
         // reduce target velocity as we get closer to home. Zero within 2m of home, reducing risk of overshooting.
         // if quad drifts further than 2m away from home, should by then have rotated towards home, so pitch is allowed
-        rescueState.intent.rollAngleLimitDeg = gpsRescueConfig()->maxRescueAngle * rescueState.intent.proximityToLandingArea;
+        rescueState.intent.rollAngleLimitDeg = 0.5f * gpsRescueConfig()->maxRescueAngle * rescueState.intent.proximityToLandingArea;
```

Comment:
What is the reason to multiply with `0.5f *` here?

Reply:
In both currently open PR's, I now limit the max roll angle to half the max pitch angle, otherwise roll can be too aggressive at times.

Reply:
Keeping values the same will mean less re-factoring before merging